controllers/bootstrap/init.go

```diff
@@ -1374,11 +1376,48 @@ func (b *Bootstrap) updateApprovalMode() error {
 		return err
 	}
 
-	if err = b.UpdateCsOpApproval(); err != nil {
+	if err = b.UpdateOpApproval(constant.IBMCSPackage); err != nil {
 		klog.Errorf("Failed to update common service operator subscription: %v", err)
```

Comment:
Hi @qpdpQ 

Could we have some hard-coded operator name replaced by variables? e.g. `constant.IBMCSPackage`